### PR TITLE
Add MV to `reporting.vw_pin_value_long`

### PR DESCRIPTION
**Pull request by wrridgeway**
Quick PR to add market value columns from `asmt_all` to the long version of `vw_pin_value`.

### CODE REVIEW

File: dbt/models/reporting/docs.md [hunk 1]
```diff
@@ -80,7 +80,10 @@ views.
 # vw_pin_value_long
 
 {% docs view_vw_pin_value_long %}
-Assessed values by PIN and year, for each assessment stage.
+Assessed and market values by PIN and year, for each assessment stage.
+
+- Market value (`_mv`) columns accurately reflect incentives, statute,
+  levels of assessment, building splits, etc.
```
Comment (dfsnow) [inline dbt/models/reporting/docs.md:86]: **issue:** This is an *assumption* and is listed in the other table as such. It should be listed as an assumption here as well.

Comment (dfsnow) [inline dbt/models/reporting/docs.md:86]: **issue:** This is an *assumption* and is listed in the other table as such. It should be listed as an assumption here as well.

File: dbt/models/reporting/schema.yml [hunk 2]
```diff
@@ -226,6 +226,13 @@ models:
           config:
             where: CAST(year AS int) < {{ var('test_qc_year_start') }}
             error_if: ">21" # as of 2024-04-01
+      - not_null:
+          name: reporting_vw_pin_value_long_tot_mv_not_null
+          column_name: tot_mv
+          config:
+            where: |
+              CAST(year AS int) < {{ var('test_qc_year_start') }} AND
+              year >= '2021'
```
Comment (dfsnow) [inline dbt/models/reporting/schema.yml:235]: **question (non-blocking):** We have the test for things >= 2021 but the values constructed for >= 2020, why the difference?

Comment (wrridgeway) [reply dbt/models/reporting/schema.yml:235]: don't we only have board values in 2020?

I can update this to address that.

File: dbt/models/default/schema/default.vw_pin_condo_char.yml [hunk 3]
```diff
@@ -104,7 +104,7 @@ models:
             (is_common_area OR is_parking_space)
             AND (char_bedrooms IS NOT NULL OR char_full_baths IS NOT NULL)
           config:
-            error_if: ">48205" # as of 2024-06-03
+            error_if: ">48206" # as of 2024-06-14
```
Comment (wrridgeway) [inline dbt/models/default/schema/default.vw_pin_condo_char.yml:107]: This feels like it's going to be an ongoing issue, now. I'll take another look next week...

Comment (dfsnow) [reply dbt/models/default/schema/default.vw_pin_condo_char.yml:107]: See #515.

**Review by dfsnow — APPROVED** [review]

**Pull request merged**